### ISSUE TITLE
При публикации на paste1c.ru исправленного варианта кода вижу старый код

### ISSUE THREAD

**Issue by tormozit** [salexdv/bsl_console#331]
Опубликовал такой код https://paste1c.ru/3ddvccc1e87q
```
Функция ВыгрузитьСвойствоКоллекцииЛкс(Знач Коллекция, Знач Свойство = "Имя", Знач ТипЭлемента = Неопределено) Экспорт
	
	Если Не ЗначениеЗаполнено(Свойство) Тогда
		Если ТипЗнч(Коллекция) = Тип("Соответствие") Тогда
			Свойство = "Ключ";
		КонецЕсли;
	КонецЕсли;
	Результат = Новый Массив;
	Структура = Новый Структура(Свойство);
	Для каждого Элемент Из Коллекция Цикл
		Если Истина
			И ТипЭлемента <> Неопределено
			И ТипЗнч(Элемент) <> ТипЭлемента 
		Тогда
			Продолжить;
		КонецЕсли;
		ЗаполнитьЗначенияСвойств(Структура, Элемент, Свойство); 
		Результат.Добавить(Структура[Свойство]);
	КонецЦикла;
	Возврат Результат;
	
КонецФункции
```
Потом исправил в нем ошибку (добавил строку) и публикую (опять новый "Поделиться")
![изображение](https://github.com/salexdv/bsl_console/assets/11896251/c4459bd7-887b-49ee-995c-0d4b3af7d9d8)

```
Функция ВыгрузитьСвойствоКоллекцииЛкс(Знач Коллекция, Знач Свойство = "Имя", Знач ТипЭлемента = Неопределено) Экспорт
	
	Если Не ЗначениеЗаполнено(Свойство) Тогда
		Если ТипЗнч(Коллекция) = Тип("Соответствие") Тогда
			Свойство = "Ключ";
		КонецЕсли;
	КонецЕсли;
	Результат = Новый Массив;
	Структура = Новый Структура(Свойство);
	Для каждого Элемент Из Коллекция Цикл
		Если Истина
			И ТипЭлемента <> Неопределено
			И ТипЗнч(Элемент) <> ТипЭлемента 
		Тогда
			Продолжить;
		КонецЕсли;
		**Структура[Свойство] = Неопределено;**
		ЗаполнитьЗначенияСвойств(Структура, Элемент, Свойство); 
		Результат.Добавить(Структура[Свойство]);
	КонецЦикла;
	Возврат Результат;
	
КонецФункции
```
Но отображается все равно старый

**Comment by salexdv** [comment]
Возможность обновлять код есть только у авторизованного пользователя. 
Верхняя панель тогда выглядит так
![image](https://github.com/salexdv/bsl_console/assets/6356534/b2b82079-6d09-4b83-be78-d6180125d35d)
Судя по скрину твоей панели, ты публикуешь под анонимом. В таком режиме код не обновляется, а формируется новая ссылка

**Comment by tormozit** [comment]
Да. Я не авторизовался. Но я не обновлял код. Кнопка "Поделиться" у меня присутствует всегда. А кнопку "Новая ссылка" я видел никогда. Обновлять код я не хотел. Я создавал новую публикацию с исправленным вариантом кода. Пробовал раз 10 - каждый раз новую ссылку получал и по ней видел старый код.

**Comment by salexdv** [comment]
Я вижу, что ты несколько раз публиковал код и он одинаковый, но воспроизвести не получилось. По новой ссылке новый код. Может есть какие-то нюансы, например, опубликовал, затем ссылку открыл другом окне/браузере или еще что-то такое?
![chrome_YXtghUWa8L](https://github.com/salexdv/bsl_console/assets/6356534/c9773856-f8a0-4faa-9b1a-2042d710f2ff)

**Comment by tormozit** [comment]
Пробовал сначала в одном бразуере. Потом несколько раз проверил в другом браузере. Похоже что это плавающая проблема на стороне сервера - он как будто из какого то кэша в тот момент брал текст.

**Comment by salexdv** [comment]
Проблема не связана с [этим](https://github.com/salexdv/bsl_console/issues/332), может не там менял код?

**Comment by tormozit** [comment]
> Проблема не связана с [этим](https://github.com/salexdv/bsl_console/issues/332), может не там менял код?

Да. Если нажать на гиперссылку "Описание", то там отобразится новый код.

**Comment by tormozit** [comment]
Получается достаточно один раз в описание вставить код и опубликовать. Потом в этом же окне если жмешь "Поделиться", то там уже не видно заголовка поля описания, а видно лишь программный код в нем. И сразу возникает ложное понимание, что новый код надо вставлять туда. 

Предлагаю всегда отображать заголовок поля, чтобы был понятен его смысл. 
Кажется я уже не первый раз спотыкаюсь об это.
![изображение](https://github.com/salexdv/bsl_console/assets/11896251/13b59012-9da1-4071-aab9-cb5a804f1540)

**Comment by salexdv** [comment]
Да, заголовка полей не видно, если они заполнены, но зачем вообще в поле "Описание" вставлять код?

**Comment by tormozit** [comment]
> зачем вообще в поле "Описание" вставлять код?

Затем, что люди ошибаются. И если они ошиблись один раз, не нужно усугублять их ошибку путем затруднения понимания что они ошиблись.
